backend/auth-service/src/main/kotlin/com/nosql/authservice/dto/SignInResponseDto.kt

```diff
@@ -0,0 +1,6 @@
+package com.nosql.authservice.dto
+
+data class SignInResponseDto (
+
```

Comment:
may be add `@JsonProperty` annotation here?

Reply:
fixed